File: solutions/beecrowd/2685/2685.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

while line := input().strip():
    degree = int(line)

    if degree == 360 or degree < 90:
        output('Bom Dia!!\n')
    elif degree < 180:
        output('Boa Tarde!!\n')
    elif degree < 270:
        output('Boa Noite!!\n')
    else:
        output('De Madrugada!!\n')

File: solutions/beecrowd/2685/2685.cpp
#include <cstdint>
#include <iostream>

int main() {
    std::ios_base::sync_with_stdio(false);
    std::cin.tie(0);

    int16_t degree;

    while (std::cin >> degree) {
        if (degree == 360 || degree < 90) {
            std::cout << "Bom Dia!!\n";
        } else if (degree < 180) {
            std::cout << "Boa Tarde!!\n";
        } else if (degree < 270) {
            std::cout << "Boa Noite!!\n";
        } else {
            std::cout << "De Madrugada!!\n";
        }
    }

    return 0;
}
